code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -640,3 +640,74 @@
 		to_chat(usr, "<span class='warning'>[src] won't wear that!</span>")
 		return
 	..()
+
+/obj/item/weapon/reagent_containers/glass/replenishing/rescue
+	name = "rescue barrel"
+	reagent_list = list(LEPORAZINE)
+
+/mob/living/simple_animal/corgi/saint
+	name = "saint corgi"
+	real_name = "saint corgi"
+	desc = "It's a saint bernard corgi mix breed. It has a tiny rescue barrel strapped around his collar to warm up travelers."
+	icon_state = "saint_corgi"
+	icon_living = "saint_corgi"
+	icon_dead = "saint_corgi_dead"
+	health = 60
+	maxHealth = 60
+	minbodytemp = 0
+	var/turns_since_scan = 0
+	var/mob/living/carbon/victim = null
+	can_breed = FALSE //tfw no gf
```

Comment:
you're a collaborator you're supposed to be teaching these young coders GOOD practices.